pkg/cmd/server/api/types.go

```diff
@@ -491,6 +491,14 @@ type HTPasswdPasswordIdentityProvider struct {
 
 type LDAPPasswordIdentityProvider struct {
 	api.TypeMeta
+	// ClientConfig is the configuration for the LDAP client connection
+	LDAPClientConfig
+	// Attributes maps LDAP attributes to identities
+	Attributes LDAPAttributeMapping
+}
+
+// LDAPClientConfig holds information for connecting to an LDAP server
+type LDAPClientConfig struct {
 	// URL is an RFC 2255 URL which specifies the LDAP search parameters to use. The syntax of the URL is
 	//    ldap://host:port/basedn?attribute?scope?filter
 	URL string
```

Comment:
The documentation around bindDN/bindPassword is also very specific to the identity provider use. The "search phase" terminology doesn't apply to sync